### DISCUSSION TITLE
Issues with Go enrichment and Tribolium IDs

### DISCUSSION THREAD

**Discussion by mprotas69** [geneontology/helpdesk#357] [General]
Hi,
I have been using the go enrichment analysis with a set of tribolium IDs that I annotated my crustacean genome with using blast2go.  Strangely, some of these IDs are showing up as unmapped IDs even though I can find them on NCBI.  For a list of around 80 genes, around 40 were shown as unmapped.  An example of two unmapped ones are NP_001139379.1 and NP_001159495.1
I can look them up in NCBI and find other codes for them: TC008204 and TC010349 which are mapped when I put them through go enrichment.
However, this is not feasible for me to do with a very large list of IDs. What do you recommend? Thanks!

**Comment by suzialeksander** [comment]
answered by PANTHER:


Unfortunately I don't know what file has the information. We use the data from UniProt and Reference Proteome. Obviously your IDs are not in their data. You mentioned that you found the TC IDs from NCBI. Maybe you can look into their data. The other possibility is to contact Gregor Bucher's group, which performed the assembly of the genome. Here is their paper https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-019-6394-6.

Best regards,

PANTHER feedback
